src/pages/playlist-page/indext.tsx

```diff
@@ -57,7 +57,11 @@ export default function PlaylistPage() {
             <div className="flex-col ml-4 mr-10">
               {activePLaylist.videos &&
                 activePLaylist.videos.map((data: any) => (
-                  <VideoCardExpanded cardData={data} playlistId={playlistId} />
+                  <VideoCardExpanded
+                    cardData={data}
+                    key={Math.random()}
```

Comment:
I wouldn't suggest using Math.random to generate key because every time a component's key changes, React will[create a new component instance rather than update the current one. (https://reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html#recommendation-fully-uncontrolled-component-with-a-key), so for performance's sake using Math.random() isn't optimum. But since this isn't a production app so you can ignore this I guess.

Reply:
Thanks, I've only used this as a workaround. I'll keep this in mind.